images/macos/toolsets/toolset-11.json

```diff
@@ -233,12 +233,18 @@
             "5.0"
         ]
     },
-    "rubygems": [
-        "xcode-install",
-        "cocoapods",
-        "nomad-cli",
-        "xcpretty",
-        "bundler",
-        "fastlane"
-    ]
+    "ruby": {
+        "default": "2.7",
+        "rubygems": [
+            "xcode-install",
+            "cocoapods",
+            "nomad-cli",
+            "xcpretty",
+            "bundler",
+            "fastlane"
+        ]
+    },
+    "go": {
+        "default": "1.15"
```

Comment:
I believe we should remove `go@1.15` from the `common_packages` section as we did for others toolsets

Reply:
thank you, fixed